install/troubleshooting/collector_hang.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Collector hangs
+backlink_href: /docs/install/troubleshooting
+backlink_title: 'Installation Troubleshooting'
+---
+
+If you find the collector hanging either periodically or constantly, or even within the test run,
+the most likely cause is `pg_stat_statements` statistic data being big and taking long time to query the statistic from it.
+This especially happens when there are many queries with long query texts.
+
+You can check out below points using the following query to identify if ps_stat_statements statistic data is big or not.
+
+* How many queries does `pg_stat_statements` statistic data currently have
+* The maximum number of queries `pg_stat_statements` keeps track of (default: 5000)
+* Total size of tracked queries
+
+```sql
+SELECT
+  count(*),
+  current_setting('pg_stat_statements.max'),
+  pg_size_pretty(sum(length(query)))
+FROM pg_stat_statements;
+```
+
+To mitigate the issue, you can run `SELECT pg_stat_statements_reset()` to reset the statistic data.
```

Comment:
```suggestion
To mitigate the issue, you can run `SELECT pg_stat_statements_reset()` to reset the statistics data.
```

I think it would also be good to mention something like "This will not affect the statistics data tracked within pganalyze, but if you are using other tools that rely on `pg_stat_statements` data, you should review how they will be affected."